tests/test_palette_c_interface.py

```diff
@@ -33,13 +33,16 @@
 
 
 class TestPalette_C_Interface(TestCase):
-    def test_init_mono(self):
+    @staticmethod
+    def test_init_mono():
```

Comment:
Like this

Reply:
Yes, `staticmethod` works for no-self-use. `classmethod` is a different thing.

Reply:
Cool. Thanks!